src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,275 @@
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmQMNfSbGS8qkJbatQgxMUsz27G8YELWgtXeLs8uFCZoY8"
+        });
+
+        // https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
+        CentrifugeCollateralValues memory RWA004 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xa92D4082BabF785Ba02f9C419509B7d08f2ef271,
+            GEM: 0x146b0abaB80a60Bfa3b4fDDb5056bBcFa4f1fec1,
+            OPERATOR: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            INPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            OUTPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            URN:  0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59,
+            gemID: "RWA004",
+            joinID: "MCD_JOIN_RWA004_A",
+            urnID: "RWA004_A_URN",
+            inputConduitID: "RWA004_A_INPUT_CONDUIT",
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilk_string: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_815_730 * WAD,
+            MAT: 11_000,
+            TAU: 0,
+            DOC: "QmYR2PXwLpdXS8Vp1yS39SPFT1XhmgbsK6XvZ9ApRpNV8M"
+        });
+
+        // https://vote.makerdao.com/polling/QmZCPDVY#poll-detail
+        CentrifugeCollateralValues memory RWA005 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x1233d0DBb55A4Bb41D711d4B584f8DDB15A2Ff88,
+            GEM: 0xcB2A48D26970eE7193d66BAc6F1b3090f2E8f82B,
+            OPERATOR: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            INPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            OUTPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            URN:  0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f,
+            gemID: "RWA005",
+            joinID: "MCD_JOIN_RWA005_A",
+            urnID: "RWA005_A_URN",
+            inputConduitID: "RWA005_A_INPUT_CONDUIT",
+            outputConduitID: "RWA005_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA005",
+            ilk: "RWA005-A",
+            ilk_string: "RWA005-A",
+            ilkRegistryName: "RWA005-A: Centrifuge: Fortunafi",
+            RATE: FOUR_PT_FIVE_PCT,
+            CEIL: 15 * MILLION,
+            PRICE: 17_199_394 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmbgDoPn6UcfSDENDqHLgatMFoqXikC8E8it9WaZXyLXmc"
+        });
+
+        // https://vote.makerdao.com/polling/QmdHV2tW#poll-detail
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x039B74bD0Adc35046B67E88509900D41b9D95430,
+            GEM: 0x4E65F06574F1630B4fF756C898Fe02f276D53E86,
+            OPERATOR: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            INPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            OUTPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
```

Comment:
I was looking through the wards on the `TinlakeManager`, and while this is a little out-of-scope for the Maker team.  When we validate, it would be nice if you could codify a list of the wards per manager, and have a test that validates those.  I think this would just be good practice so that we can double check for you, and it shows us that you thought about those authorizations.

You have two wards on each of these AFAICT, one is the `TinlakeRoot`, the other is the `clerk`?  Looks like this is it, but it's unverified: https://kovan.etherscan.io/address/0xec8ff575e916e95188547dfb346d9d5fda97453b#code

Reply:
Good point. I've just verified `0xEc8FF575e916e95188547DFB346d9d5fDa97453b`, this is indeed the clerk, which is the internal Maker adapter on the Tinlake side. On mainnet, the clerk won't be deployed yet in most cases (for pools which we're integrating which were already live before the maker integration) though. We'll do this in an integration spell on our side after the executive vote.

Checking at least that the `TinlakeRoot` is a ward on the manager is definitely a good idea, since if that's missing we're basically in a deadlock state. I'll add a simple test for this.

Reply:
Just added a test for this: https://github.com/offerijns/spells-kovan/blob/fb6301675d29bcad49f621dd079797cda9a547f0/src/Kovan-DssSpell.t.sol#L1854